README.md

```diff
@@ -21,6 +21,7 @@ For the setup, it is assumed that there are multiple LND nodes running with conn
   * LND nodes configuration: TLS certificate, macaroon, address and pubkey. Pubkey is configured as a protection against unintentionally connecting to the wrong node.
   * Postgres connection string for the database created above (can be overridden by the environment variable `LNMUX_PERSISTENCE_POSTGRES_DSN` to avoid exposing credentials).
   * 32-byte Identity PRIVATE key. This key is used to generate invoices and decode incoming htlcs. In a production environment, this key must be protected carefully (can be overridden by the environment variable `LNMUX_IDENTITY_KEY` to avoid exposing the private key).
+  * Your logging configuration. You can choose the level, the encoding format, to display grpc logs or not,...
```

Comment:
"or not,..." ?

Reply:
Maybe it is unclear, I will remove the 'or not'

Reply:
Ah, you mean "etcetera"? Maybe more informative to write it as something like "the most important options are x and y".

Reply:
Yeah, I mean etcetera, but you are right, your formulation looks better!